.github/workflows/stale-issues-workflow.yml

```diff
@@ -52,5 +55,5 @@ jobs:
           # do not manage PRs
           process-only: issues
           # stale issue config
-          issue-lock-inactive-days: 0 # immediately lock closed issues
+          issue-lock-inactive-days: 0  # immediately lock closed issues
```

Comment:
extra whitespace?

Reply:
Yeah, yamllint said it was necessary 🤷‍♂️